src/systemctl/systemd-sysv-install.SKELETON

```diff
@@ -11,6 +11,8 @@ usage() {
     exit 1
 }
 
+ROOT=
```

Comment:
shouldn't this be "unset ROOT"?

Reply:
It doesn't really matter. Usually `[ -n "$ROOT" ]` would be used, since checking if a variable is defined isn't the most common thing in shell.

Reply:
Well, sure, but outside of the shell it's kinda weird having an env var stick around that is assigned the empty string. I mean, this var is inherited to all kinds of processes forked off from this script, and they might act on it too, and find it set to some really strange value...

Reply:
hm, for it to leak into the called processes, wouldn't ROOT need to be exported, which is not the case her?

Reply:
but wasn't the original problem that the script might have inherited and only changed it in some cases but passed it on in others?